docs/source/osds/scenarios.rst

```diff
@@ -1,12 +1,232 @@
+
+To use this configuration, the ``lvm_volumes`` option must be populated with
+logical volumes, volume groups, and optionally partitions for journals, or
+``block.db`` and ``block.wal``.
+
+.. note:: This configuration uses ``ceph-volume lvm create`` to provision OSDs
+
+Supported ``lvm_volumes`` configuration settings:
+
+- ``data``: The logical volume name or full path to a raw device (an LV will be
+  created using 100% of the raw device)
+
+- ``data_vg``: The volume group name. If using a logical volume name for ``data``
+    it is **required**.
+
+- ``crush_device_class``: Device class name for the resulting OSD, allows to
+  individually set the class for each OSD, unlike the global
+  ``crush_device_class`` that sets them for all OSDs.
+
+.. note:: If you wish to change set the ``crush_device_class`` for the OSDs
+          when using ``devices`` you must set it using the global ``crush_device_class``
+          option as shown above. There is no way to define a specific crush device class
+          per OSD when using ``devices`` like there is for ``lvm_volumes``.
+
+
+``filestore`` objectstore only values:
+
+- ``journal``: The logical volume name or full path to a partition.
+
+- ``journal_vg``: The volume group name. If using a logical volume name for
+  ``journal`` it is **required**.
+
+
+``bluestore`` objectstore only values:
+
+- ``db``: The logical volume name or full path to a partition.
+
+- ``db_vg``: The volume group name. If using a logical volume name for ``db``
+    it is **required**.
+
+- ``wal``: The logical volume name or full path to a partition.
+
+- ``wal_vg``: The volume group name. If using a logical volume name for ``wal``
+    it is **required**.
+
+
+.. note:: These ``bluestore`` settings are optional, and are useful if placing
+          them in faster devices. It is possible to create a bluestore OSD with
+          a single raw device.
+
+.. warning:: It is not allowed to use the same values for many OSDs (for
+             example, a journal LV cannot be used for more than one OSD)
+
+``bluestore`` example using raw devices only:
+
+.. code-block:: yaml
+
+   osd_objectstore: bluestore
+   osd_scenario: lvm
+   lvm_volumes:
+     - data: /dev/sda
+     - data: /dev/sdb
+
+
+``bluestore`` example with logical volumes only:
+
+.. code-block:: yaml
+
+   osd_objectstore: bluestore
+   osd_scenario: lvm
+   lvm_volumes:
+     - data: data-lv1
+       data_vg: data-vg1
+     - data: data-lv2
+       data_vg: data-vg2
+
+
+``bluestore`` example defining ``wal`` and ``db`` logical volumes:
+
+.. code-block:: yaml
+
+   osd_objectstore: bluestore
+   osd_scenario: lvm
+   lvm_volumes:
+     - data: data-lv1
+       data_vg: data-vg1
+       db: db-lv1
+       db_vg: db-vg1
+       wal: wal-lv1
+       wal_vg: wal-vg1
+     - data: data-lv2
+       data_vg: data-vg2
+       db: db-lv2
+       db_vg: db-vg2
+       wal: wal-lv2
+       wal_vg: wal-vg2
+
+
+``filestore`` example with logical volumes only:
```

Comment:
the 'only' seems to not carry much information but a bit of confusion, and could be elided